security/voters.rst

```diff
@@ -31,6 +31,22 @@ in the application, which can be: affirmative, consensus or unanimous.
 For more information take a look at
 :ref:`the section about access decision managers <components-security-access-decision-manager>`.
 
+.. tip::
+
+    The example used throughout this page features just two routes (``post_show`` and ``post_edit``).
+    However, the advantage of voters is that you can reuse them in *many* places and centralize
+    all permission logic. If you don't reuse permissions or the rules are basic, you instead
+    might want to do the check in the controller directly and throw an ``AccessDeniedException``
+    to create the correct response::
+
+        // src/AppBundle/Controller/PostController.php
+        use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+
+        // ...
+        if ($post->getOwner() !== $this->getUser()) {
+            throw new AccessDeniedException();
```

Comment:
Let's use: `throw $this-> createAccessDeniedException()` - we always assume users are using the normal shortcuts - like `AbstractController`.

Reply:
Done :-)